doc/specs/top_10.md

```diff
@@ -0,0 +1,106 @@
+# Top 10
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+
+## Motivation
+
+Users are notified of the top 10 most popular albums on the website. Users can view the list in a homepage widget/module. Users can see the most popular albums by past day, past week, past month, and all-time.
+
+## Business rules
+
+- Users can enable or disable the top 10 list from displaying on their homepage.
+- Each item in the list must display the album's title, artist, and release year.
+- The top album must also display the album cover art.
+- The list will be automatically updated periodically (12-24 hours?) by a ranking algorithm.
+
+## Use cases
+
+### 1.0 - Enabling the Top 10
+
+**Actor:** A user.
+
+**Brief:**  The Top 10 is enabled for display on the actor's homepage.
+
+**Trigger:** The actor wants to enable the Top 10.
+
+**Preconditions:**
+
+ - The actor has the Top 10 widget/module disabled in their settings.
+
+**Postconditions:**
+
+ - The Top 10 widget/module is enabled and shown on the actor's homepage.
+
+**Basic Flow:**
+
+ 1. The actor navigates to their settings.
+ 2. The actor toggles Top 10 from disabled to enabled.
+ 3. The actor saves their settings.
+
+### 1.1 - Disabling the Top 10
```

Comment:
You can merge 1.0 and 1.1 into "Modifying the top 10 widget", or something along those liens.